### ISSUE TITLE
Add reader_method to GlobReaderGenerator

### ISSUE THREAD

**Issue by cwerner** [fivetran/great_expectations#914]
I use `read_method: csv` and `known_extensions: [.dat]` with the `SubdirReaderGenerator`. However, if I want to use the `GlobReaderGenerator` these options are not allowed.

To get around this I currently modify the batch_kwargs in the code:

```python
batch_kwargs = context.yield_batch_kwargs(data_asset_name)
batch_kwargs = dict(batch_kwargs)
batch_kwargs['reader_method'] = 'csv'
batch_kwargs['known_extensions'] = ['.dat']
```

**Comment by jcampbell** [comment]
Adding support for a `reader_method` key to GlobReaderGenerator seems like a really great improvement, and would a constrained PR, if you'd be willing to add it!

Since with `GlobReaderGenerator` you have full control over the globs to match, you do already specify desired extensions using the glob pattern. In the example you gave, as long as the glob pattern ends in `.dat`, you'll get the desired behavior: known_extensions does *not* need to be included in the `batch_kwargs`

**Comment by cwerner** [comment]
I submitted the PR (#915) - not sure if me CLA was registered?

**Comment by jcampbell** [comment]
Looks like Kyle updated it--thank you!!

**Comment by cwerner** [comment]
Closing as this was just merged in #915
